apps/token/src/routes/governance/components/vote-details/vote-details.tsx

```diff
@@ -30,6 +31,42 @@ export const VoteDetails = ({ proposal }: VoteDetailsProps) => {
     requiredParticipation,
   } = useVoteInformation({ proposal });
 
+  let minVoterBalance = null;
```

Comment:
question (blocking): How does spam protection play into this? Do we need to take a max?